grammar.js

```diff
@@ -345,12 +345,12 @@ module.exports = grammar({
         ),
 
         string_literal: $ => seq(
-            alias(/b?"/, '"'),
+            token('"'),
```

Comment:
I might be mistaken, but running `let foo = b"test"` does not work with my version of nu. Am I overlooking something?

Reply:
You are right. I stole a bunch of the current stuff from the rust tree sitter. It's a great resource that we can copy from.